src/assets/Generator.Shared/CodeGenMemberSerializationHooksAttribute.cs

```diff
@@ -26,5 +32,10 @@ internal class CodeGenMemberSerializationHooksAttribute : Attribute
         public CodeGenMemberSerializationHooksAttribute()
         {
         }
+
+        public CodeGenMemberSerializationHooksAttribute(string propertyName)
```

Comment:
is it necessary to have this constructor?

Reply:
This is for property which is not in this class but inherit from base class

Reply:
yes, I mean we can just have parameterless constructor since `PropertyName` has a setter already.
Why we have this constructor is because the user can just write `[CodeGenMemberSerializationHooks(nameof(Id),...)` instead of `[CodeGenMemberSerializationHooks(PropertyName: nameof(Id),...)`.